src/components/post-ctrls/RepostButton.tsx

```diff
@@ -35,33 +39,16 @@ let RepostButton = ({
   const {_, i18n} = useLingui()
   const requireAuth = useRequireAuth()
   const dialogControl = Dialog.useDialogControl()
-  const playHaptic = useHaptics()
-  const color = React.useMemo(
-    () => ({
-      color: isReposted ? t.palette.positive_600 : t.palette.contrast_500,
-    }),
-    [t, isReposted],
-  )
+
   return (
     <>
-      <Button
+      <PostCtrlButton
         testID="repostBtn"
-        onPress={() => {
-          playHaptic('Light')
-          requireAuth(() => dialogControl.open())
-        }}
-        onLongPress={() => {
-          playHaptic('Heavy')
-          requireAuth(() => onQuote())
-        }}
-        style={[
-          a.flex_row,
-          a.align_center,
-          a.gap_xs,
-          a.bg_transparent,
-          {padding: 5},
-        ]}
-        hoverStyle={t.atoms.bg_contrast_25}
+        active={isReposted}
+        activeColor={t.palette.positive_600}
+        big={big}
+        onPress={() => requireAuth(() => dialogControl.open())}
+        onLongPress={() => requireAuth(() => onQuote())}
         label={
           isReposted
             ? _(
```

Comment:
```suggestion
        label={
          isReposted
            ? _(
                msg({
                  message: `Undo repost (${plural(repostCount || 0, {
                    one: '# repost',
                    other: '# reposts',
                  })})`,
                  comment: 'Accessibility label for the repost button when the post has been reposted, verb form followed by number of reposts and noun form',
                })
              )
            : _(
                msg({
                  message: `Repost (${plural(repostCount || 0, {
                    one: '# repost',
                    other: '# reposts',
                  })})`,
                  comment: 'Accessibility label for the repost button when the post has not been reposted, verb form followed by number of reposts and noun form',
                })
              )
        }
```
And I also couldn't get it to select all the lines here, either